From 77cbde31f25d625b9ed91bc37ca65bf4bb026582 Mon Sep 17 00:00:00 2001
From: Chris Chiu <chiu@endlessm.com>
Date: Thu, 2 May 2019 18:10:22 +0800
Subject: [PATCH 28/29] hwdb: Align airplane mode toggle key mapping for all
 Acer series

Packard Bell and Gateway are different marketing names from Acer.
The same scan code E0 86 is fired for the airplane mode toggle key.
It was verified in commit d8d51328fe6db33a2d8cda06f181c55c00d09672.

Signed-off-by: Chris Chiu <chiu@endlessm.com>
(cherry picked from commit 82b941cb5d1fa04b6c13dba4f07af4be8b21561e)
---
 hwdb/60-keyboard.hwdb | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/hwdb/60-keyboard.hwdb b/hwdb/60-keyboard.hwdb
index b74ff4cdc1..15505b96ea 100644
--- a/hwdb/60-keyboard.hwdb
+++ b/hwdb/60-keyboard.hwdb
@@ -182,8 +182,9 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnAspire*1640:*
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnAcer*:pnAOA*:pvr*
  KEYBOARD_KEY_a9=!switchvideomode                       # Fn+F5
 
-# Easynote models
-evdev:atkbd:dmi:bvn*:bvr*:bd*:svnPackard*Bell*:pnEasynote*:pvr*
+# Packard Bell and Gateway models
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnGateway*:pn*
+evdev:atkbd:dmi:bvn*:bvr*:bd*:svnPackard*Bell*:pn*
  KEYBOARD_KEY_86=wlan                                   # Fn+F3 or Fn+Q for comunication key
 
 ###########################################################
-- 
2.21.0

